frontend/plugins/frontline_ui/src/modules/integrations/discord/components/DiscordIntegrationDetail.tsx

```
@@ -179,10 +259,10 @@ export const DiscordIntegrationDetail = () => {
         });
       }
 
+      void refetchNamePresets();
       setOpen(false);
       reset();
     } catch {
-      // error toast is surfaced by useIntegrationAdd
     }
```

ESLint

[error] Empty block statement.

(no-empty)